src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let totalCost = days * 40;
+
+  if (days >= 7) {
+    totalCost -= 50;
```

Comment:
you can return result right away and then you don't need to change value of totalCost
```suggestion
    return totalCost - 50;
```